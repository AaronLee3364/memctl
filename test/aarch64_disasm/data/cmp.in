fffffff0070c0000 f100393f 6b08017f 6b2821df
