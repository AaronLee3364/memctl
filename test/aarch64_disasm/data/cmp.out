fffffff0070c0000  CMP     X9, #0xe
fffffff0070c0004  CMP     W11, W8
fffffff0070c0008  CMP     W14, W8, UXTH
